elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -31,7 +31,7 @@ $nav-variables: (
 
   // Trigger region
   trigger: (
-    FontSize: 16px,
+    FontSize: #{pfe-var(font-size)},
```

Comment:
```suggestion
    FontSize: pfe-var(font-size),
```

Reply:
@starryeyez024 I think this is a good update but we will need to fix our theme to customize this in the scope of the pfe-navigation so that we continue to have 16px triggers.

Reply:
@dcaryll Do you foresee an issue using the font-size base (which in PatternFly and PatternFly Elements is 16px) on the navigation trigger?  That base is 18px on dotcom but we could customize that for navigation.  Should the trigger ever be larger or smaller than 16px?  Do we need to hardcode it for design system purposes?

Reply:
What if rather than using a heading size like epsilon, it might be easier if the pfe-nav used the default body copy size which is already 16px.  Then on redhat.com where our default boy copy is 18px, we can include an override in our theme which changes the pfe-nav-trigger vars to 16px?

Reply:
@starryeyez024 I'm not sure what you mean, can you clarify?  Where are you seeing epsilon in use?  This update is using the font-size variable which is the body copy variable.  My comment was meant to communicate that we will need to customize the body copy variable in the scope of pfe-navigation for dotcom if this update is made.

Reply:
I see epsilon used in the shared_assets.scss in this variable: `$trigger--FontSize--fallback`.  That variable is being used in the lightdom file but I think it's missing the scoping it needs to ensure that is only applied on Edge/older browsers.